test/polyfill.v0.js

```diff
@@ -0,0 +1,4 @@
+import 'webcomponents.js/src/ShadowDOM/ShadowDOM';
+import 'webcomponents.js/src/WeakMap/WeakMap';
+import 'webcomponents.js/src/MutationObserver/MutationObserver';
+import 'webcomponents.js/src/CustomElements/v1/CustomElements';
```

Comment:
You'll also want to check to see if you need those other polyfills with it (I just remember possibly not needing them).